src/components/Footer.tsx

```diff
@@ -79,7 +79,7 @@ export const Footer: React.FC = () => {
                 <Link to="/docs/getting-started/install">Getting started</Link>
               </li>
               <li>
-                <a href="https://github.com/mswjs/examples">Examples</a>
+                <Link to="/examples">Examples</Link>
```

Comment:
I noticed this link points to the examples repo.
Is that intended? If so, I can revert this change.

Reply:
That's a good catch. Since now we have a dedicated Examples page, let it point to it, as you've corrected.